cpp/JsiWorkletContext.cpp

```diff
@@ -61,7 +60,7 @@ void JsiWorkletContext::initialize(
   _contextId = ++contextIdNumber;
 
   _jsThreadId = std::this_thread::get_id();
-  runtimeMappings.emplace(&getWorkletRuntime(), this);
+  runtimeMappings.emplace(&_workletRuntime, this);
```

Comment:
This needs to be `getWorkletRuntime()` otherwise _workletRuntime is an invalid pointer (the getWorkletRuntime() initialises the worklet runtime.